base/testfiles-ltcmd/github-1009.tlg

```diff
@@ -4,32 +4,30 @@ Don't change this file in any respect.
 TEST 1: Non-expandable Document Commands with m-type arguments
 ============================================================
 > \foo=document command:
+ #1:m
 ->.
```

Comment:
`\ShowCommand` works wrong.

- This `\foo` takes no arguments, hence shouldn't show `#1:m`.
- For `\foo` that does take `m` arguments, `\__cmd_start_optimised: .` is shown.

Reply:
I realise looking at this we do need a two-stage process - which I'd missed. Update coming up.

Reply:
Tricker than expected: I need to re-work the test for `ltcmd` support entirely

Reply:
OK, I have this right in `ltcmd` I think but there is still more to do :(

Reply:
Should now be correct